sdk/servicebus/azure-servicebus/azure/servicebus/aio/_async_message.py

```diff
@@ -129,7 +129,7 @@ async def defer(self) -> None:  # type: ignore
         await self._settle_message(MESSAGE_DEFER)
         self._settled = True
 
-    async def renew_lock(self) -> datetime.datetime:
+    async def renew_lock(self, timeout=None) -> datetime.datetime:
```

Comment:
Is this a public API? If so this should either be in `kwargs` or using keyword-only syntax `(self, *, timeout=None)`

Reply:
moved into kwargs, thanks for the call out